<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_msg">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>明文</string>
    </property>
    <widget class="QPlainTextEdit" name="messageText">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>301</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_cip">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>密文</string>
    </property>
    <widget class="QPlainTextEdit" name="cipherText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_param">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>参数</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="text_k">
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_vi">
        <property name="text">
         <string>初始向量vi</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="text_vi">
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_k">
        <property name="text">
         <string>初始密钥k</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_encode">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;加密&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_decode">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;解密&lt;&lt;&lt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
